src/main/java/build/buildfarm/server/BuildFarmServer.java

```diff
@@ -89,10 +121,17 @@ public BuildFarmServer(String session, ServerBuilder<?> serverBuilder, BuildFarm
             TimeUnit.SECONDS,
             keepaliveScheduler))
         .addService(new OperationQueueService(instances))
-        .addService(new OperationsService(instances))
+        .addService(new OperationsService(instances));
+
+    if (!tokens.isEmpty()) {
```

Comment:
I suggest we don't do it...
there should always be auth. and if there is no token, the server shouldn't load (k8s will not load the pod anyway if it can't mount all requested secrets.)

Reply:
Recall that this is also used for local development where auth should be disabled.